docker/integration/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+flux start python3 -m pytest $(python3 -c \
+	"import parsl.tests.test_flux as tf; print(tf.__file__)") \
+	--config=local --tap-stream
```

Comment:
For my own edification, what does `--config=local` do?

Reply:
That's a parsl flag they added to `pytest`, the way you set up Parsl is you define a `config`, and the flag is basically to specify that I don't need a special site-specific config.